ci-jobs/templates/ios-e2e-template.yml

```diff
@@ -42,7 +43,7 @@ jobs:
       condition: eq('${{ parameters.launchWithIDB }}', true)
     - task: NodeTool@0
       inputs:
-        versionSpec: 10.x
+        versionSpec: 12.x
```

Comment:
Will this repository use 12 instead of minimal our support nodejs version?

Reply:
@KazuCocoa I just wanted to bump it up to 12 because that's the LTS

Reply:
The unit tests cover 10 and 12. I think the typical coverage is to get the LTS and the most recent version before it.